includes/payments/functions.php

```diff
@@ -1832,3 +1832,31 @@ function edd_filter_where_older_than_week( $where = '' ) {
 
 	return $where;
 }
+
+/**
+ * Gets the payment ID from the final edd_payment post.
+ * This was set as an option when the custom orders table was created.
+ * For internal use only.
+ *
+ * @return false|int
+ */
+function edd_get_final_payment_id() {
```

Comment:
This is another great candidate for an `_` function. We are labeling it for `internal use only`, so let's prefix it in a common way that identifies that. Which will make it easier for us to remove later as well.

While we are doing this issue, please make an issue for 3.1 that mentions all of these methods/functions so we can mark them as deprecated in 3.1. If we log them all now, we have less of a chance that we'll forget about them and end up maintaining them long term.